legal-api/src/legal_api/reports/report.py

```diff
@@ -748,6 +751,18 @@ def _format_amalgamation_data(self, filing):
     def _format_certificate_of_amalgamation_data(self, filing):
         self._set_amalgamating_businesses(filing)
 
+    def _format_notice_of_withdrawal_data(self, filing):
+        withdrawn_filing_id = filing['noticeOfWithdrawal']['filingId']
+        withdrawn_filing = Filing.find_by_id(withdrawn_filing_id)
+        formatted_withdrawn_filing_type = self._format_filing_type(withdrawn_filing.filing_type)
+        filing['withdrawnFilingType'] = formatted_withdrawn_filing_type
+        withdrawn_filing_date = LegislationDatetime.as_legislation_timezone(withdrawn_filing.effective_date)
+        filing['withdrawnFilingEffectiveDate'] = LegislationDatetime.format_as_report_string(withdrawn_filing_date)
+
+    def _format_filing_type(self, filing_type_str: str):
+        words = ''.join(' ' + c if c.isupper() else c for c in filing_type_str).split()
+        return ' '.join(word.capitalize() if word != 'agm' else word.upper() for word in words)
```

Comment:
This func is not required. Use `FilingMeta.get_display_name` instead https://github.com/bcgov/lear/blob/main/legal-api/src/legal_api/core/meta/filing.py#L783

Reply:
Updated. Thanks for pointing it out.